test/witan/models/dem/ccm/components_functions_test.clj

```diff
@@ -61,3 +61,12 @@
       (is (every? #(fp-equals? (i/sel joined-averages :rows % :cols :intin)
                                (i/sel joined-averages :rows % :cols :international-in) 0.0000000001)
                   (range (first (:shape joined-averages))))))))
+
+(deftest make-trend-key-test
+  (testing "key returned with -trend appended "
+    (is (= :foo-trend (make-trend-key :foo)))))
+
+(deftest jumpoffyr-method-trend-test
+  (testing ""
+    (let [clj-results (jumpoffyr-method-trend (:domestic-in-migrants migration-data)
+                                                :estimate :domestic-in 2003 2014)])))
```

Comment:
Please, add a comment in the code to say it's a WIP.
Are you planning to get to that next or is it for later?